### PR TITLE
feat: make ceramic admin key configurable

### PR DESCRIPTION
**Pull request by smrz2001**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,42 @@ Inspect the pods within the network using:
...
+
+```yaml
+apiVersion: "keramik.3box.io/v1alpha1"
+kind: Network
+metadata:
+  name: small
+spec:
+  replicas: 2
+  ceramic:
+    privateKeySecret: "private-key"
```
Comment (nathanielc) [inline README.md:101]: should this be `small` as that is the name of the secret?

File: README.md [hunk 2]
```diff
@@ -81,6 +81,42 @@ Inspect the pods within the network using:
...
+using the name of the new secret in the network configuration.
+```yaml
+---
+namespace: keramik
+
+secretGenerator:
+- name: small
+  envs:
+  - .env.secret
+```
```
Comment (nathanielc) [inline README.md:119]: Could you add an example of the `.env.secret` file?

**Review by nathanielc — APPROVED** [review]
Looks great, with a small fix to the README merge away.

File: README.md (hunk 1 above)
Comment (smrz2001) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (smrz2001) [reply README.md:127]: And done!

**Comment by smrz2001** [comment]
> Looks great, with a small fix to the README merge away.

Thanks so much for your help, @nathanielc!!!

**Pull request merged**